osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -113,8 +117,21 @@ private void load(OsuConfigManager config, NotificationOverlay notificationOverl
             ModDisplay.Current.Value = mods;
 
             showHud = config.GetBindable<bool>(OsuSetting.ShowInterface);
-            showHud.ValueChanged += visible => visibilityContainer.FadeTo(visible.NewValue ? 1 : 0, duration);
-            showHud.TriggerChange();
+            showHud.BindValueChanged(visible => visibilityContainer.FadeTo(visible.NewValue ? 1 : 0, duration, easing), true);
+
+            ShowHealthbar.BindValueChanged(healthBar =>
```

Comment:
we can just retrieve the bindable from the config just like `showHud`, but that means we'll most likely revert all mod changes

Reply:
i'm fine with leaving this for now. we'd have to iterate mods locally in `HUDOverlay`, which seems maybe bad.

Reply:
yeah, that way is better